test/unit/puppetlabs/puppetserver/ruby/http_client_test.clj

```diff
@@ -262,59 +266,64 @@ jruby-config :- jruby-schemas/JRubyConfig
+          (with-scripting-container sc
+            (with-http-client sc {}
+              (let [url (str "https://localhost:10080")]
+                (.runScriptlet sc (format "$response = $c.get('%s')" url))
+                (is (= 200 (.runScriptlet sc "$response.code")))
+                (is (= "hi" (.runScriptlet sc "$response.body"))))))))
+
+      (testing "Can connect via TLSv1.1 by default"
+        (with-webserver-with-protocols ["TLSv1.1"] ["TLS_RSA_WITH_AES_128_CBC_SHA"]
+          (with-scripting-container sc
+            (with-http-client sc {}
+              (.runScriptlet sc (format "$response = $c.get('%s')" url))
+              (is (= 200 (.runScriptlet sc "$response.code")))
+              (is (= "hi" (.runScriptlet sc "$response.body")))))))))
 
   (testing "Can connect via TLSv1.2 by default"
     (with-webserver-with-protocols ["TLSv1.2"] nil
       (with-scripting-container sc
-        (with-http-client sc 10080 {:use-ssl true}
-          (.runScriptlet sc "$response = $c.get('/', {})")
-          (is (= "200" (.runScriptlet sc "$response.code")))
-          (is (= "hi" (.runScriptlet sc "$response.body"))))))))
+        (with-http-client sc {}
+          (let [url (str "https://localhost:10080")]
+            (.runScriptlet sc (format "$response = $c.get('%s')" url))
+            (is (= 200 (.runScriptlet sc "$response.code")))
+            (is (= "hi" (.runScriptlet sc "$response.body")))))))))
 
 (deftest clients-persist
   (testing "client persists when making HTTP requests"
     (logutils/with-test-logging
       (jetty9/with-test-webserver ring-app port
-      (with-scripting-container sc
-        (with-http-client sc port {:use-ssl false}
-          (let [client1 (.runScriptlet sc "$c.get('/', {}); $c.class.client")
-                client2 (.runScriptlet sc "$c.post('/', 'foo', {}); $c.class.client")]
-            (is (= client1 client2))))))))
+        (with-scripting-container sc
+          (with-http-client sc {}
+            (let [url (str "http://localhost:" port)
+                  client1 (.runScriptlet sc (format "$c.get('%s'); $c.class.client" url))
+                  client2 (.runScriptlet sc (format "$c.post('%s', 'foo'); $c.class.client" url))]
+              (is (= client1 client2))))))))
   (testing "all instances of HttpClient have the same underlying client object"
     (logutils/with-test-logging
       (jetty9/with-test-webserver ring-app port
         (with-scripting-container sc
-          (with-http-client sc port {:use-ssl false}
+          (with-http-client sc {}
             (let [client1 (.runScriptlet sc "$c.class.client")
-                  client2 (.runScriptlet sc (str (format "$c2 = Puppet::Server::HttpClient.new('localhost', %d, {:use_ssl => %s});"
-                                                         port false)
+                  client2 (.runScriptlet sc (str (format "$c2 = Puppet::Server::HttpClient.new;"
+                                                         false)
```

Comment:
I don't think this `false` is needed anymore. Looks like it was the `:use_ssl` value.